src/components/Footer/footer.scss

```diff
@@ -0,0 +1,42 @@
+@import "../../variables.scss";
+
+.footer-container {
+  display: flex;
+  height: 24px;
+  padding: 16px 24px;
+  justify-content: space-between;
+  align-items: center;
+  background: $actions-primary-active
+}
+
+.footer-container__left {
+  display: flex;
+  align-items: center;
+  gap: 24px;
+
+  a {
+    color: $backgrounds-content-2;
+    font-family: Roboto;
```

Comment:
the font family is set in project level css file, please don't introduce redundant styling